tests/standalone-tests/pom.xml

```diff
@@ -82,7 +82,10 @@
             <artifactId>parsson</artifactId>
             <scope>test</scope>
         </dependency>
-
+        <dependency>
+            <groupId>org.wildfly.channel</groupId>
+            <artifactId>channel-core</artifactId>
```

Comment:
This looks like it should have `<scope>test</scope>`.

Reply:
Good, fixed.